state-chain/custom-rpc/src/lib.rs

```diff
@@ -280,6 +280,21 @@ pub struct RpcEnvironment {
 	pools: PoolsEnvironment,
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
+pub struct PoolPriceV2 {
+	pub base_asset: RpcAsset,
+	pub quote_asset: RpcAsset,
+	pub buy_price: Option<PoolPriceV1>,
+	pub sell_price: Option<PoolPriceV1>,
```

Comment:
would it make sense to add the range order price (PRO-1050) here? @AlastairHolmes